### ISSUE TITLE
prisma error/ Timeout for query 

### ISSUE THREAD

**Issue by osnapitsjoey** [imagegenius/docker-ai-chat-app#18]
`ready - started server on 0.0.0.0:3000, url: http://localhost:3000
prisma:error 
Invalid `prisma.template.upsert()` invocation:

Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: ConnectionError(Timed out during query execution.), transient: false })
prisma:error 
Invalid `prisma.chat.findUniqueOrThrow()` invocation:

An operation failed because it depends on one or more records that were required but not found. Expected a record, found none.
prisma:error 
Invalid `prisma.chat.findUniqueOrThrow()` invocation:

An operation failed because it depends on one or more records that were required but not found. Expected a record, found none.
prisma:error 
Invalid `prisma.chat.findUniqueOrThrow()` invocation:

An operation failed because it depends on one or more records that were required but not found. Expected a record, found none.
prisma:error 
Invalid `prisma.chat.findUniqueOrThrow()` invocation:

An operation failed because it depends on one or more records that were required but not found. Expected a record, found none`

**Comment by osnapitsjoey** [comment]
apologize if this has something to do with the other thing posted today

**Comment by osnapitsjoey** [comment]
update: 

still getting 

`prisma:error 
Invalid `prisma.template.upsert()` invocation:


Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: ConnectionError(Timed out during query execution.), transient: false })
`
unraid btw. seems connection times out immediately

**Comment by hydazz** [comment]
What are your variables

**Comment by osnapitsjoey** [comment]
webui
NEXTAUTH_SECRET
NEXTAUTH_URL
ADMIN_EMAIL
OPEN_AI_KEY

DATABASE_URL=file:/config/db.sqlite
PUID=99
CHARSET=UTF-8
HOSTNAME=##########
S6_CMD_WAIT_FOR_SERVICES_MAXTIME=0
HOST_CONTAINERNAME=ai-chat-app
NEXTAUTH_URL=http://192.168.1.2:3000
UMASK=022
PWD=/
NODE_ENV=production
TZ=America/Los_Angeles
ADMIN_EMAIL=####################################
HOME=/root
LANG=C.UTF-8
PGID=100
ADMIN_PASSWORD=########################
S6_VERBOSITY=1
S6_STAGE2_HOOK=/docker-mods
OPEN_AI_KEY=###########################################
NEXTAUTH_SECRET=##############################
TERM=xterm
HOST_OS=Unraid
SHLVL=1
PS1=$(whoami)@$(hostname):$(pwd)\$ 
LC_COLLATE=C
PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
HOST_HOSTNAME=Server
_=/usr/bin/env

edited out personal info

**Comment by hydazz** [comment]
Try deleting the appdata folder (db.sqlite)

**Comment by osnapitsjoey** [comment]
Tried that. Deleted it every time we had an update incase that was the issue

**Comment by hydazz** [comment]
Do the errors occur as soon as the container starts?

**Comment by osnapitsjoey** [comment]
Seems they appear after trying to get chat to function, or import a
template

**Comment by hydazz** [comment]
is `NEXTAUTH_URL` the actual IP of unraid?

**Comment by osnapitsjoey** [comment]
No, it's just the local address. I will try the actual address though and
let you know

**Comment by osnapitsjoey** [comment]
okay just reinstalled and am now able to save templates to saved, but chat still throws

prisma:error 
Invalid `prisma.chat.findUniqueOrThrow()` invocation:


An operation failed because it depends on one or more records that were required but not found. Expected a record, found none.
prisma:error 
Invalid `prisma.chat.findUniqueOrThrow()` invocation:


An operation failed because it depends on one or more records that were required but not found. Expected a record, found none.

all of my set-up is just how it should be and i have change my lan ip to correspond to nextauth instead of http://[ip]:[port]

**Comment by hydazz** [comment]
I believe this is caused by the API rejecting requests, but I'm unsure, I setup billing and now it works...
